scale/README.md

```diff
@@ -242,15 +242,19 @@ below for reference.
 | ENABLE_BOOTSTRAP            | 'true'                          | Bootstrap Scale support containers         |
 | ENABLE_WEBSERVER            | 'true' or None                  | Used by bootstrap to enable UI and API     |
 | FLUENTD_DOCKER_IMAGE        | 'geoint/scale-fluentd'          | Docker image for fluentd                   |
+| FLUENTD_BUFFER_WARN         | 10                              | Warn if buffer exceeds this many items     |
```

Comment:
Could we get these alphabetized? `FLUENTD_BUFFER_*` should go before `FLUENTD_DOCKER_IMAGE` and `MESSSAGE_QUEUE_DEPTH_WARN` after `MESOS_ROLE`.

Reply:
👍